### PR TITLE
FLUT-829266 - [Others] Updated the Flutter DataGrid document.

### PR DESCRIPTION
**Pull request by Tamilarasan-Paranthaman**
## Description ##

* I have updated the Flutter DataGrid UG.

### CODE REVIEW

File: Flutter/datagrid/swiping.md [hunk 1]
```diff
@@ -154,23 +154,23 @@ class EmployeeDataSource extends DataGridSource {
...
-* [onSwipeUpdate](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/onSwipeUpdate.html): Called while swiping a row is in progress. The swipe action can be canceled by return `false`.
-* [onSwipeEnd](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/onSwipeEnd.html): called when the swipe offset value reaches the `SfDataGrid.maxSwipeOffset` indicating that the swipe action is completed.
+* [onSwipeStart](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/onSwipeStart.html): Called when the swipe offset changes from its initial value. The swipe action can be canceled by returning `false`.
+* [onSwipeUpdate](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/onSwipeUpdate.html): Called while swiping a row is in progress. The swipe action can be canceled by returning `false`.
+* [onSwipeEnd](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/onSwipeEnd.html): called when the swipe offset value reaches the `SfDataGrid`.maxSwipeOffset` indicates that the swipe action is completed.
```
Comment (Chrispineagunja) [inline Flutter/datagrid/swiping.md:165]: * [onSwipeEnd](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/onSwipeEnd.html): Called when the swipe offset value reaches the `SfDataGrid`.maxSwipeOffset` indicates that the swipe action is completed.

File: Flutter/datagrid/swiping.md [hunk 2]
```diff
@@ -154,23 +154,23 @@ class EmployeeDataSource extends DataGridSource {
...
 
 The swipe callbacks provide the following properties in their arguments:
 
 * [RowIndex](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/DataGridSwipeUpdateDetails/rowIndex.html): Defines the swiping row index.
 * [SwipeDirection](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/DataGridSwipeUpdateDetails/swipeDirection.html): Defines the swipe direction of the swiped row.
 * [SwipeOffset](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/DataGridSwipeUpdateDetails/swipeOffset.html): Defines the current swipe offset of the row being swiped.
 
-By handling the swipe callbacks, you can use these properties value from the arguments to perform any desired action such as deleting the row, editing the data, etc.
+By handling the swipe callbacks, you can use these property values from the arguments to perform any desired action such as deleting the row, editing the data, etc.
```
Comment (Chrispineagunja) [inline Flutter/datagrid/swiping.md:173]: By handling the swipe callbacks, you can use these property values from the arguments to perform any desired action, such as deleting the row, editing the data, and more.

File: Flutter/datagrid/swiping.md [hunk 3]
```diff
@@ -260,11 +260,11 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-![flutter datagrid shows customized swiping delete functionality](images/swiping/flutter-datagrid-customized-swiping-delete-funtionality.gif)
+![flutter Datagrid shows customized swiping delete functionality](images/swiping/flutter-datagrid-customized-swiping-delete-funtionality.gif)
 
 ## Set different swipe offsets for right and left swiping 
 
-You can set the different swipe offsets based on swipe direction by using the `onSwipeStart` callback and passing the required swipe offset to the [setSwipeMaxOffset](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/DataGridSwipeStartDetails/setSwipeMaxOffset.html) method from the `onSwipeStart` callback's argument.  
+You can set the different swipe offsets based on swipe direction by using the `onSwipeStart` callback and passing the required swipe offset to the [setSwipeMaxOffset](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/DataGridSwipeStartDetails/setSwipeMaxOffset.html) method from the `onSwipeStart` callback argument.
```
Comment (Chrispineagunja) [inline Flutter/datagrid/swiping.md:267]: Set the different swipe offsets based on swipe direction by using the `onSwipeStart` callback and passing the required swipe offset to the [setSwipeMaxOffset](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/DataGridSwipeStartDetails/setSwipeMaxOffset.html) method from the `onSwipeStart` callback argument.

**Comment by Edberg-Vasan** [comment]
rebuild

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine.

**Review by FarjanaParveen — APPROVED** [review]
This is fine

**Pull request merged**